biocrnpyler/species.py

```diff
@@ -627,7 +634,7 @@ def __init__(self, species, name = None, material_type = "ordered_complex", attr
 
         #Call the Species superclass constructor
         Species.__init__(self, name = name, material_type = material_type, attributes = attributes, 
-                        compartment = compartment, initial_concentration = initial_concentration)
+                            compartment = compartment, initial_concentration = initial_concentration)
```

Comment:
tab space

Reply:
I used a autopep8 extension on this file `species.py` to fix this and other similar possible issues.. Let me know if it looks fine.